docs/features/star-search.md

```diff
@@ -12,6 +12,11 @@ keywords:
 - "open source maintainers" 
 - "AI"
 ---
+::: tip
+
+Currently, StarSearch is in development, but you can [sign up for the waitlist now](https://app.opensauced.pizza/star-search/waitlist).
+
+:::
```

Comment:
This doesn't render well on website as below:

![tip](https://github.com/open-sauced/docs/assets/45172775/127243da-c5fb-45fb-8dd8-40ad32bdf65c)

Let's add a new line after `---` for best practice and remove the space before `tip` to fix the admonition.

```suggestion
---

:::tip

Currently, StarSearch is in development, but you can [sign up for the waitlist now](https://app.opensauced.pizza/star-search/waitlist).

:::
```